### ISSUE TITLE
While integrating lib-raw-data in android the callback method of MediaPreprocessing class i.e. onCaptureVideoFrame, onPreEncodeVideoFrame or onRenderVideoFrame are not executed for getting video frame of LocalVideoStreaming

### ISSUE THREAD

**Issue by TriveniRajput** [AgoraIO/API-Examples#140]

**Comment by yoreland** [comment]
do you reproduce this issue by run the demo directly?

**Comment by Jack29913** [comment]
I have the same issue, doesn't work in demo

**Comment by plutoless** [comment]
@Jack29913 it might be you failed to join the channel. has your project turned on token feature?

**Comment by yoreland** [comment]
@Jack29913 @TriveniRajput 
can you please check branch dev/android
should be just fixed. c++ headers are updated to 3.3.1 version

**Comment by TriveniRajput** [comment]
@yoreland yes i have directly run it from demo..  for this  c++ headers are updated to 3.3.1 version I will check it.

**Comment by Jack29913** [comment]
@plutoless no, after the modification both clients successfully joined the channel. we don't and won't use token feature.

**Comment by yoreland** [comment]
The fix just merged to master branch.
My local ndk version is 21, we will do some ndk compatible test and add them to readme doc.
Thanks for the information!

**Comment by ashimneupane** [comment]
@Jack29913 @yoreland Running into the same issue with version 3.3.1. How do I get the 2.9.4 version? Could you provide the commit id?
